flink-staging/flink-ml/src/main/scala/org/apache/flink/ml/classification/SVM.scala

```diff
@@ -232,7 +281,13 @@ object SVM{
 
   implicit def predictVectors[T <: Vector] = {
```

Comment:
Can we add some scala docs? It's actually my fault because it was already missing when I merged the change of the predict operation.